oauth2client/contrib/appengine.py

```diff
@@ -49,9 +49,19 @@
 # TODO(dhermes): Resolve import issue.
 # This is a temporary fix for a Google internal issue.
 try:
-    from google.appengine.ext import ndb
```

Comment:
As currently drafted (so on the right side of the diff; sorry that this conversation is happening on the left): aren't the names `CredentialsNDBProperty`, `FlowNDBProperty`, and `SiteXsrfSecretKeyNDB` intended for application use?